services/consumer/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+	"time"
+
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v1"
+	"github.com/red-hat-storage/ocs-operator/controllers/storagecluster"
+	"github.com/red-hat-storage/ocs-operator/controllers/util"
+	providerclient "github.com/red-hat-storage/ocs-operator/services/provider/client"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+func main() {
+	flag.Parse()
+
+	scheme := runtime.NewScheme()
+	if err := ocsv1.AddToScheme(scheme); err != nil {
+		klog.Exitf("failed to add ocsv1 to scheme: %v", err)
+	}
+
+	config, err := config.GetConfig()
+	if err != nil {
+		klog.Exitf("failed to get config: %v", err)
+	}
+	client, err := client.New(config, client.Options{Scheme: scheme})
+	if err != nil {
+		klog.Exitf("failed to create controller-runtime client: %v", err)
+	}
+	namespace, err := util.GetWatchNamespace()
```

Comment:
We should change this according to my other comment about the env in the external_ocs_heartbeat.go

Reply:
Updated in latest commit